spec/apollo-server-spec.yaml

```diff
@@ -0,0 +1,56 @@
+swagger: "2.0"
+info:
+  description: "This is the backend component API. Visit the link for an interactive playground to execute requests."
```

Comment:
nit: which link?

Reply:
Oh yeah, it supposedly refers to the url where the backend service is exposed. But for now I'll avoid that part of the description since it's not relevant now.